### PR TITLE
Roselia LM fixed

### PR DESCRIPTION
**Pull request by Grayahm**
Fixed Roselia

### CODE REVIEW

File: src/tcgwars/logic/impl/gen3/LegendMaker.groovy [hunk 1]
```diff
@@ -1611,12 +1611,12 @@ public enum LegendMaker implements LogicCardInfo {
           onAttack {
             def pcs = defending
             if (opp.bench) {
-              if (confirm("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon.")) {
-                pcs = opp.bench.oppSelect()
+              if (confirm("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon?")) {
```
Comment (flagrama) [inline src/tcgwars/logic/impl/gen3/LegendMaker.groovy:1614]: The text implies this should not be a choice.

Comment (Grayahm) [reply src/tcgwars/logic/impl/gen3/LegendMaker.groovy]: Fixed

Comment (flagrama) [reply src/tcgwars/logic/impl/gen3/LegendMaker.groovy]: No, I mean you shouldn't use a `confirm` here. The text should go on the `select` instead, and the confirm should be removed.

Comment (Grayahm) [reply src/tcgwars/logic/impl/gen3/LegendMaker.groovy]: Oh there was more wrong with this card than I thought haha. That should do it.

**Comment by luongthomas** [comment]
Changes look good, but I believe you need to rebase your branch

File: src/tcgwars/logic/impl/gen3/LegendMaker.groovy [hunk 2]
```diff
@@ -1610,13 +1610,12 @@ public enum LegendMaker implements LogicCardInfo {
...
           onAttack {
             def pcs = defending
-
-            if (opp.bench && confirm("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon.")) {
-              pcs = opp.bench.select("Select a Benched Pokémon to become the new Active Pokémon")
-              sw defending, pcs
+            if (opp.bench) {
+                pcs = opp.bench.select("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon.")
+                sw defending, pcs
+              }
```
Comment (flagrama) [inline src/tcgwars/logic/impl/gen3/LegendMaker.groovy:1616]: I sees it.

**Pull request merged**